hr_holidays_public/models/hr_leave.py

```diff
@@ -8,6 +8,17 @@
 class HrLeave(models.Model):
     _inherit = "hr.leave"
 
+    def action_validate(self):
```

Comment:
This shouldn't be in this commit, but cherry picking the original commit

Reply:
Squashing I wrongly merged both commits cherry-picked and my work ... will look to see if I find a way of fixing it or I will create again the PR

Reply:
This is still pending

Reply:
@pedrobaeza with my last PR

changes you commented are now chery-picked in commit

ca9063544ef691b9d14eab0bdb0102fc1e72ec77

my changes are in commit

c815f50c7277d0e42620ff34bafbcdd0eb980f26